model/facade/TasksFacade.php

```diff
@@ -369,6 +370,24 @@ static function GetGlobalUsersProjectsReport(DateTime $init = NULL, DateTime $en
 
     }
 
+    /** Get weekly hours worked per week on a given project
+     *
+     * This function is used to retrieve details of hours worked by all users on a given project
+     * <var>$projectVO</var>
+     *
+     * @param ProjectVO $projectVO
+     * @param DateTime|null $init
+     * @param DateTime|null $end
+     * @return mixed
+     * @throws null
+     */
+    static function GetProjectUserWeeklyHoursReport(ProjectVO $projectVO, DateTime $init = NULL, DateTime $end = NULL) {
+
+    $action = new GetProjectUserWeeklyHoursReportAction($projectVO, $init, $end);
+
+    return $action->execute();
```

Comment:
The indentation is wrong in the rest of the file, but let's do it right for new methods.